src/index.js

```diff
@@ -67,10 +66,13 @@ async function runner(ctx, name) {
 		for (hook of before) await hook();
 		for (test of arr) {
 			try {
+				for (hook of bEach) await hook();
 				await test.handler();
+				for (hook of aEach) await hook();
 				write(PASS);
 				num++;
 			} catch (err) {
+				for (hook of aEach) await hook();
```

Comment:
Sorry for the delay – I was thinking about it, and this is all the "additional cleanup" that was needed.

Reply:
Nice!